.github/PR_TEMPLATE/PULL_REQUEST_CP_TEMPLATE.md

```diff
@@ -4,18 +4,26 @@ Please fill in the form below, and a flutter domain expert will evaluate this ch
 ### Issue Link:
 What is the link to the issue this cherry-pick is addressing?
 
-< Replace with issue link here >
-
-### Changelog Description:
-Explain this cherry pick in one line that is accessible to most Flutter developers. See [best practices](https://github.com/flutter/flutter/blob/main/docs/releases/Hotfix-Documentation-Best-Practices.md) for examples
-
-< Replace with changelog description here >
+https://github.com/flutter/flutter/issues/< Replace with issue link here >
 
 ### Impact Description:
-What is the impact (ex. visual jank on Samsung phones, app crash, cannot ship an iOS app)? Does it impact development (ex. flutter doctor crashes when Android Studio is installed), or the shipping production app (the app crashes on launch)
+What is the impact (ex. visual jank on Samsung phones, app crash, cannot ship an iOS app)?
+Does it impact development (ex. flutter doctor crashes when Android Studio is installed),
+or the shipping production app (the app crashes on launch).
```

Comment:
Nit, I know you didn't write this:

"or the shipping production app" => "or the shipping of production apps"

I think that is more the intention?